Compound_Risk_database.R

```diff
@@ -40,44 +42,182 @@ normfuncpos <- function(df, upperrisk, lowerrisk, col1) {
 #
 ##
 ### ********************************************************************************************
-####    CREATE HEALTH SHEET USING A RANGE OF SOURCE INDICATORS
+####    CREATE ACAPS SHEET USING A RANGE OF SOURCE INDICATORS ----
 ### ********************************************************************************************
 ##
 #
 
-#--------------------HIS Score-----------------
+#--------------------—LOAD ACAPS realtime database-------------------------------------------
```

Comment:
I moved ACAPS section to front because it is included in other indicator sheets. (`Global summary ...R` no longer reads from ACAPS sheet directly.)